packages/ast/src/util/JSON.ts

```diff
@@ -0,0 +1,43 @@
+import { Precision } from '../types';
+import { Expression } from '../nodes';
+import { NodeID } from './Node';
+import { NodeTagValue } from './Tags';
+import { u32 } from './types'; 
+import { TypeID } from '../types/Type';
+
+export interface iJSON {
```

Comment:
Do not use this interface, this is just like using a non typed JSON.

Reply:
done in [changed ast imports, deleted JSON interface] commit